projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/interestrate/future/derivative/InterestRateFutureOptionMarginSecurity.java

```diff
@@ -5,33 +5,13 @@
  */
 package com.opengamma.analytics.financial.interestrate.future.derivative;
 
-import org.apache.commons.lang.ObjectUtils;
-
 import com.opengamma.analytics.financial.interestrate.InstrumentDerivativeVisitor;
 import com.opengamma.util.ArgumentChecker;
-import com.opengamma.util.money.Currency;
 
 /**
  * Description of an interest rate future option with up-front margin security.
```

Comment:
change description to "Description of an interest rate future option security with daily margin process."